Extensions/Today/Info.plist

```diff
@@ -2,8 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>MozGroupSuffix</key>
-	<string>$(MOZ_GROUP_SUFFIX)</string>
+	<key>AppIdentifierPrefix</key>
+	<string>$(APP_IDENTIFIER_PREFIX)</string>
```

Comment:
I don't like having `APP_IDENTIFIER_PREFIX` as a user defined property since it's confusing with `AppIdentifierPrefix` which is used in the entitlements and contains a period at the end of the team ID where as this does not contain the period. Technically this is just the DEVELOPMENT_TEAM property but Ill introduce that change in another patch.